packages/client/hooks/useQueryLoaderNow.tsx

```diff
@@ -1,23 +1,48 @@
 import areEqual from 'fbjs/lib/areEqual'
-import {useEffect, useRef} from 'react'
-import {PreloadableConcreteRequest, PreloadFetchPolicy, useQueryLoader} from 'react-relay'
+import {useCallback, useEffect, useRef} from 'react'
+import {
+  fetchQuery,
+  PreloadableConcreteRequest,
+  PreloadFetchPolicy,
+  useQueryLoader,
+  useRelayEnvironment
+} from 'react-relay'
 import {GraphQLTaggedNode, OperationType, VariablesOf} from 'relay-runtime'
 import useAtmosphere from './useAtmosphere'
 
 const useQueryLoaderNow = <TQuery extends OperationType>(
   preloadableRequest: GraphQLTaggedNode | PreloadableConcreteRequest<TQuery>,
   variables: VariablesOf<TQuery> = {},
-  fetchPolicy?: PreloadFetchPolicy
+  fetchPolicy?: PreloadFetchPolicy,
+  preventSuspense?: boolean
 ) => {
   const [queryRef, loadQuery] = useQueryLoader<TQuery>(preloadableRequest)
   const varRef = useRef(variables)
   const atmosphere = useAtmosphere()
+  const environment = useRelayEnvironment()
   if (!areEqual(variables, varRef.current)) {
     varRef.current = variables
   }
+
+  const refreshQuery = useCallback(() => {
```

Comment:
This logic prevents the results from suspending on each keystroke, which causes the modal to flash. It could probably be cleaned up a little - I don't like the name `refreshQuery` for example when we have `refresh` lower down in the file